core/src/test/java/tech/tablesaw/columns/ColumnTest.java

```diff
@@ -193,9 +195,34 @@ public void testFilter() {
         Column<Double> filtered = DoubleColumn.create("t1", new double[] {-1, 0, 1}).filter(isPositiveOrZero);
         check(filtered, 0.0, 1.0);
     }
+	private String getSeason(LocalDate date) {
+		String season = "";
+		int month = date.getMonthValue();
+		int day = date.getDayOfMonth();
+				
+		if ( (month == 1) || (month == 2) 
```

Comment:
please don't add parentheses around every individual condition. It's not consistent with this library's coding style

Reply:
Will do.  Thanks.